### PR TITLE
Api test com_privacy

### PR DESCRIPTION
**Pull request by andyforrest**
Summary of Changes

API tests for following endpoints:

- GET: /api/index.php/v1/privacy/requests

- GET: /api/index.php/v1/privacy/requests/{request_id}

- POST: /api/index.php/v1/privacy/requests

missing:

- GET: /api/index.php/v1/privacy/request/export/{request_id}

### CODE REVIEW

File: tests/System/support/commands/db.js [hunk 1]
```diff
@@ -42,6 +42,23 @@ function createInsertQuery(table, values) {
 
   return query;
 }
+/**
```
Comment (laoneo) [inline tests/System/support/commands/db.js]: Can you add here a new line?

File: tests/System/integration/api/com_privacy/Request.cy.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      .then((response) => cy.api_responseContains(response, 'email', 'test@example.com'));
+  });
+  it('can list a single request', () => {
+    cy.db_createPrivacyRequest()
+      .then((id) => cy.api_get(`/privacy/requests/${id}`))
+      .then((response) => cy.wrap(response).its('body').its('data').its('attributes')
+        .its('status')
+        .should('equal', 0));
+  });
+  it('can create a request', () => {
```
Comment (laoneo) [inline tests/System/integration/api/com_privacy/Request.cy.js]: Can you add here a new line?

File: tests/System/integration/api/com_privacy/Request.cy.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+describe('Test privacy request API endpoint', () => {
+  afterEach(() => cy.task('queryDB', 'DELETE FROM #__privacy_requests'));
+  it('can get a list of requests', () => {
+    cy.db_createPrivacyRequest()
+      .then(() => cy.api_get('/privacy/requests'))
+      .then((response) => cy.api_responseContains(response, 'email', 'test@example.com'));
+  });
+  it('can list a single request', () => {
```
Comment (laoneo) [inline tests/System/integration/api/com_privacy/Request.cy.js]: Can you add here a new line?

File: tests/System/integration/api/com_privacy/Request.cy.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+describe('Test privacy request API endpoint', () => {
+  afterEach(() => cy.task('queryDB', 'DELETE FROM #__privacy_requests'));
+  it('can get a list of requests', () => {
```
Comment (laoneo) [inline tests/System/integration/api/com_privacy/Request.cy.js]: Can you add here a new line?

**Comment by laoneo** [comment]
Why did you close?

**Comment by andyforrest** [comment]
> Why did you close?

Sorry I deleted the branch on github by mistake! I made a new PR with ammendments

**Pull request closed without merging**